tests/ppx/main_deriving.ml

```diff
@@ -0,0 +1,16 @@
+module Type_conv = Ppx_type_conv.Type_conv
+
+let l = Ppx_deriving.derivers ()
+let _ = ListLabels.iter l ~f:(fun (deriver : Ppx_deriving.deriver) ->
+  Type_conv.add deriver.Ppx_deriving.name
```

Comment:
The feature is implemented and released in our tree, it will be on github next time we push, so hopefully next week